host/3.0/buster/amd64/base/host.Dockerfile

```diff
@@ -1,5 +1,5 @@
 
-ARG HOST_VERSION=3.3.1
+ARG HOST_VERSION=3.4.0
```

Comment:
We don't get some of the benefits at this point because of the existence of v2upgrade and non-appservice images. But, in the future this script will only need to update this location in order to create and run our scripts.